js/src/blocks/product-select-with-text-field/edit.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { useWooBlockProps } from '@woocommerce/block-templates';
+import { SelectControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import {
+	__experimentalUseProductEntityProp as useProductEntityProp,
+	__experimentalTextControl as TextControl,
+} from '@woocommerce/product-editor';
+
+/**
+ * Internal dependencies
+ */
+import { Label } from '../components';
+
+const FALLBACK_VALUE = '';
+
+export default function Edit( { attributes, context } ) {
```

Comment:
💅  Can we add a bit of documentation of what `customInputValue` attribute is for?

Reply:
Added JSDoc for all custom blocks by https://github.com/woocommerce/google-listings-and-ads/pull/2178/commits/d8252b8b9e6214a6223f7fef139a75c267fb4e81.